package it.`is`.all.good.coupon.exception

class CouponNotFoundException(message: String) : RuntimeException(message)
